docs/topics/proc_adding-source-platform-application.adoc

```diff
@@ -0,0 +1,46 @@
+:_newdoc-version: 2.15.0
+:_template-generated: 2024-2-21
+:_mod-docs-content-type: PROCEDURE
+
+[id="adding-source-platform-application_{context}"]
+= Adding a source platform in an application
+
+[role="_abstract"]
+You can associate applications that you created in {ProductShortName} with their Cloud Foundry (CF) instances by configuring the application's coordinates in the source platform. 
+
+The specific coordinates vary depending on the source platform you select. For example, the coordinates for CF instances are *name* and *space*. 
+
+.Prerequisites
+
+* You configured at least one source platform instance in the *Administration* mode.
+
+.Procedure
+
+. In the {ProductShortName} user interface, select the *Migration* working mode.
+. Click *Application Inventory* in the left menu bar.
+. Click the edit icon (image:icon_edit.png[]). The *Update application* dialog appears.
+. Enter the following information in the form:
+. Click *Source platform* and enter the following fields:
+.. *Source platform*: Select a source platform instance.
+.. *Source platform coordinates*: Enable the source platform coordinates to enter application coordinates in JSON. For example, a CF instance coordinates can be:
++
+[source, json]
+----
+{
+    "names": ["app-a", "app-b"],
+    "spaces": ["example-space"]
+}
```

Comment:
In the app instance the source platform coordinates are the name and the space of the app deployed on CF. It should not be a list of apps/spaces
```suggestion
{
  "name": "example-app",
  "space": "example-space"
}
```